File: ProAgil-App/src/app/_models/Palestrante.ts
import { RedeSocial  } from './RedeSocial';
import { Evento  } from './Evento';

export interface Palestrante {
   id: number;
   nome: string;
   resumo: string;
   imagemUrl: string;
   telefone: string;
   email: string;
   redesSociais: RedeSocial[];
   palestrantesEventos: Evento[] ;
}

File: ProAgil.Domain/Palestrante.cs
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Palestrante
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public int Resumo { get; set; }
        public int ImagemUrl { get; set; }
        public int Telefone { get; set; }
        public int Email { get; set; }
        public List<RedeSocial> RedesSociais { get; set; }
        public List<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}